source/deploy/desktop/desktop-overview.rst

```diff
@@ -0,0 +1,21 @@
+Desktop App deployment
+=======================
+
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+    :titlesonly:
+
+    /deploy/desktop/desktop-app.rst
+    /deploy/desktop/desktop-msi-installer-and-group-policy-install.rst
+    /deploy/desktop/desktop-app-managed-resources.rst
+    /deploy/desktop/desktop-custom-dictionaries.rst
+
+The Mattermost desktop application is available for Windows, macOS, and Linux operating systems. Learn about installation, configuration, and management options for deploying the desktop app in your environment.
+
+This documentation provides guidance on deploying and managing the Mattermost Desktop application across your organization.
```

Comment:
All these detailed guides are great, but I wonder if we also should have a "vanilla" one.

What should an admin do if they want to roll out the desktop in their org? What installers exists?

Reply:
I agree